src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,11 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * @return true for command types that mutate the data, returns false for commands that don't
+     */
```

Comment:
Good job remembering to add a header comment to this method. 👍 
It can be shortened as follows without losing any info. 
```java
/**
  * Returns true if the command potentially mutates data
  */
```